x-pack/plugins/security_solution/server/lib/detection_engine/signals/rule_status_saved_objects_client.ts

```diff
@@ -31,7 +31,9 @@ export const ruleStatusSavedObjectsClientFactory = (
 ): RuleStatusSavedObjectsClient => ({
   find: (options) =>
     savedObjectsClient.find<IRuleStatusAttributes>({ ...options, type: ruleStatusSavedObjectType }),
-  create: (attributes) => savedObjectsClient.create(ruleStatusSavedObjectType, attributes),
-  update: (id, attributes) => savedObjectsClient.update(ruleStatusSavedObjectType, id, attributes),
-  delete: (id) => savedObjectsClient.delete(ruleStatusSavedObjectType, id),
+  create: (attributes) =>
+    savedObjectsClient.create(ruleStatusSavedObjectType, attributes, { refresh: false }),
+  update: (id, attributes) =>
+    savedObjectsClient.update(ruleStatusSavedObjectType, id, attributes, { refresh: false }),
+  delete: (id) => savedObjectsClient.delete(ruleStatusSavedObjectType, id, { refresh: false }),
```

Comment:
Did you want the the behavior of the default `wait_for` to be changed to a `refresh: false` here? What parts of the code need this because things are too slow?

Should we just add this as an option to be passed in for just those parts of the code rather than all of them?

Reply:
The create's and updates were taking around 1 second to complete for some reason. Using `refresh: false` it dropped to like 200ms. Spent quite some time debugging this but couldn't find a resolution outside of this. I'll look into adding it as an option.

Reply:
The docs outline why its up to 1 second to complete:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html

The 1 second is very intentional due to the nature of how ES updates its documents on writes for performance. This can be configurable and we have tested with it before:

```ts
Wait for the changes made by the request to be made visible by a refresh before replying.
This doesn’t force an immediate refresh, rather, it waits for a refresh to happen.
Elasticsearch automatically refreshes shards that have changed every
index.refresh_interval which defaults to one second.
```

In the past when we didn't use `wait_for` in our rest routes what would happen is we would create a document and return but the front end would then do a query and not have any results which is why we would implement the `wait_for` so that we can guarantee the document is there when the REST route returned which is why I ask if you're sure about forcing the `false` in these areas.

Reply:
It was taking on average 3 seconds for the rules table to load and I wasn't able to find a way to speed up the creates / updates without setting the refresh to false. Not sure what else to do at this point.